src/main/resources/com/google/api/codegen/java/sample.snip

```diff
@@ -14,136 +14,133 @@
      * 3. Install the Java client library on Maven or Gradle. Check installation
      *    instructions at https://github.com/google/google-api-java-client.
      *    On other build systems, you can add the jar files to your project from
-     *    https://developers.google.com/resources/api-libraries/download/{@sampleClass.apiName}/{@sampleClass.apiVersion}/java
+     *    https://developers.google.com/resources/api-libraries/download/{@class.apiName}/{@class.apiVersion}/java
    @default
      * 2. Install the Java client library on Maven or Gradle. Check installation
      *    instructions at https://github.com/google/google-api-java-client.
      *    On other build systems, you can add the jar files to your project from
-     *    https://developers.google.com/resources/api-libraries/download/{@sampleClass.apiName}/{@sampleClass.apiVersion}/java
+     *    https://developers.google.com/resources/api-libraries/download/{@class.apiName}/{@class.apiVersion}/java
    @end
    */
-  @join import : sampleClass.imports
+  @join import : class.imports
     import {@import};
   @end
 
-  public class {@sampleClass.className} {
-    {@runFunc(sampleClass.body)}
+  public class {@class.className} {
+    {@mainFunc(class.body)}
 
-    {@createServiceFunc(sampleClass.body)}
-
-    {@mainFunc(sampleClass)}
+    {@createServiceFunc(class.body)}
   }
 @end
 
-@private runFunc(sampleBody)
-  public void run() throws IOException, GeneralSecurityException {
-    @join field : sampleBody.fields if sampleBody.fields
+@private mainFunc(body)
+  public static void main(String args[]) throws IOException, GeneralSecurityException {
+    @join field : body.fields if body.fields
       @if field.description
         @join line : util.getDocLines(field.description, 100)
           // {@line}
         @end
 
 
       @end
-      {@field.typeName} {@field.name} = {@field.defaultValue}; {@placeholderTodo(field.isPlaceholderSingular)}
+      {@field.typeName} {@field.name} = {@field.defaultValue};  {@placeholderTodo(field.isPlaceholderSingular)}
 
     @end
-    @if sampleBody.hasInputRequest
-      // TODO: Assign values to desired fields of `{@sampleBody.requestBodyVarName}`.
-      @if sampleBody.methodVerb == "PATCH"
+    @if body.hasInputRequest
+      // TODO: Assign values to desired fields of `{@body.requestBodyVarName}`.
+      @if body.methodVerb == "PATCH"
         // Only assigned fields will be changed.
       @end
-      {@sampleBody.requestBodyTypeName} {@sampleBody.requestBodyVarName} = new {@sampleBody.requestBodyTypeName}();
+      {@body.requestBodyTypeName} {@body.requestBodyVarName} = new {@body.requestBodyTypeName}();
 
     @end
-    @if sampleBody.hasMediaUpload
+    @if body.hasMediaUpload
       // TODO: Add desired media content for upload. For more information, see:
```

Comment:
To match less verbose style elsewhere, may want to remove `For more information`.